css/home-not-logged-in.styl

```diff
@@ -354,8 +354,11 @@ $flex-section
 
             h3, p
               a
-                text-decoration: none
                 color: inherit
+                display: block
```

Comment:
This is actually used for both components, so I can probably have the styling added to `on-home-page` instead, since I've now added it to the root.